src/pkg/backup/details/details.go

```diff
@@ -233,9 +268,21 @@ type DetailsEntry struct {
 	RepoRef   string `json:"repoRef"`
 	ShortRef  string `json:"shortRef"`
 	ParentRef string `json:"parentRef,omitempty"`
+
+	// LocationRef contains the logical path structure by its human-readable
+	// display names.  IE:  If an item is located at "/Inbox/Important", we
+	// hold that string in the LocationRef, while the actual IDs of each
+	// container are used for the RepoRef.
+	// LocationRef only holds the container values, and does not include
+	// the metadata prefixes (tenant, service, owner, etc) found in the
+	// repoRef.
```

Comment:
If we intend to store only a subset of the data in the reporef, how do we intend to deal with selector comparisons on that metadata for `Reduce()`? Part of this could be me being unfamiliar with the selectors package, but if we currently have selectors doing something like `entry.RepoRef.Category() == sel.category` then won't we end up splitting things in a bit of an awkward fashion? Seems like some comparisons would end up against the LocationRef while others would use the RepoRef

We don't need a full implementation right now, and I realize that may be difficult given the main thing to address at the moment is Calendars, but it's good to have an idea of where we plan to go

Reply:
If/when you look at the next PR, I think this will get cleared up.